src/components/VideoList/VideoList.tsx

```diff
@@ -37,33 +41,43 @@ function VideoList({
   accessModel,
   isLoggedIn,
   hasSubscription,
+  hasLoadMore,
+  loadMore,
 }: Props) {
-  return (
-    <div className={classNames(styles.container, !!className && className)} data-testid={testId('video-list')}>
-      {!!header && header}
-      {playlist &&
-        playlist.playlist.map((playlistItem: PlaylistItem) => {
-          const { mediaid, title, duration, seriesId, episodeNumber, seasonNumber, shelfImage } = playlistItem;
+  const List = () => {
+    return (
+      <>
+        {playlist?.playlist?.map((playlistItem: PlaylistItem) => {
+          const { mediaid } = playlistItem;
```

Comment:
I think with just 1 prop, we can omit the local variable and use `playlistItem.mediaid` where needed. It keeps the markup cleaner and closer to the html that will be rendered.
```suggestion
```

Reply:
Fixed it.